src/sentry/models/user.py

```diff
@@ -114,7 +116,6 @@ class User(BaseModel, AbstractBaseUser):
     session_nonce = models.CharField(max_length=12, null=True)
 
     date_joined = models.DateTimeField(_("date joined"), default=timezone.now)
-    last_active = models.DateTimeField(_("last active"), default=timezone.now, null=True)
```

Comment:
References to this attribute would need to be cleaned up, as evidenced by the many test failures. (Assuming that's within the scope of the exercise. I was out last time.)

Also, what is the motive for removing this? Does it outweigh the potential value of having the data later, in case we want it after all?